WooCommerce/WooCommerceTests/ViewRelated/HubMenu/HubMenuViewModelTests.swift

```diff
@@ -26,7 +26,9 @@ final class HubMenuViewModelTests: XCTestCase {
         let viewModel = HubMenuViewModel(siteID: sampleSiteID, featureFlagService: featureFlagService)
 
         // Then
-        XCTAssertFalse(viewModel.menuElements.contains(.inbox))
+        XCTAssertNil(viewModel.menuElements.firstIndex(where: { item in
+            type(of: item).id == HubMenuViewModel.Inbox.id
+        }))
```

Comment:
When we assert here, isn't `item` already using the type `HubMenuItem` implicitly? 

```
XCTAssertNil(viewModel.menuElements.firstIndex(where: { item in
    item.id == HubMenuViewModel.Inbox.id
}))
```

Do we need to declare `type(of: item).id` ?

Reply:
Great point there! I changed it in d5f725618153cd5d1d317690dfba5bb43d4045b0